--2025-03-13 23:25:11--  https://github.com/zonzorp/COMP2137/raw/main/makecontainers.sh
Resolving github.com (github.com)... 140.82.113.3
Connecting to github.com (github.com)|140.82.113.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://raw.githubusercontent.com/zonzorp/COMP2137/main/makecontainers.sh [following]
--2025-03-13 23:25:11--  https://raw.githubusercontent.com/zonzorp/COMP2137/main/makecontainers.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 15446 (15K) [text/plain]
Saving to: ‘/home/sudip/makecontainers.sh’

     0K .......... .....                                      100% 4.96M=0.003s

2025-03-13 23:25:11 (4.96 MB/s) - ‘/home/sudip/makecontainers.sh’ saved [15446/15446]

WARNING:root:Cannot call Open vSwitch: ovsdb-server.service is not running.
Traceback (most recent call last):
  File "/usr/sbin/netplan", line 23, in <module>
    netplan.main()
  File "/usr/share/netplan/netplan/cli/core.py", line 56, in main
    self.run_command()
  File "/usr/share/netplan/netplan/cli/utils.py", line 243, in run_command
    self.func()
  File "/usr/share/netplan/netplan/cli/commands/apply.py", line 63, in run
    self.run_command()
  File "/usr/share/netplan/netplan/cli/utils.py", line 243, in run_command
    self.func()
  File "/usr/share/netplan/netplan/cli/commands/apply.py", line 258, in command_apply
    subprocess.check_call(['udevadm', 'trigger', '--attr-match=subsystem=net'])
  File "/usr/lib/python3.10/subprocess.py", line 364, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python3.10/subprocess.py", line 347, in call
    return p.wait(timeout=timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1209, in wait
    return self._wait(timeout=timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1959, in _wait
    (pid, sts) = self._try_wait(0)
  File "/usr/lib/python3.10/subprocess.py", line 1917, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt
assignment2.sh: Permission denied
